packages/fxa-auth-server/lib/push.js

```diff
@@ -36,42 +72,13 @@ const TTL_COMMAND_RECEIVED = TTL_PASSWORD_CHANGED;
 // Currently only for metrics purposes, not enforced.
 const MAX_ACTIVE_DEVICES = 200;
 
-const pushReasonsToEvents = (() => {
-  const reasons = [
-    'accountVerify',
-    'accountConfirm',
-    'passwordReset',
-    'passwordChange',
-    'deviceConnected',
-    'deviceDisconnected',
-    'profileUpdated',
-    'devicesNotify',
-    'accountDestroyed',
-    'commandReceived',
-  ];
-  const events = {};
-  for (const reason of reasons) {
-    const id = reason.replace(/[A-Z]/, (c) => `_${c.toLowerCase()}`); // snake-cased.
-    events[reason] = {
-      send: `push.${id}.send`,
```

Comment:
These "push.${reason}.${outcome}` names remind me of datadog, where we had to cram things into the event name because of restrictions on tags. I think all of our current tooling can handle a simpler operation name with the reason etc as tags, and they were mildly annoying to work with in bigquery. So, I've changed them. I don't think we had anything depending on the precise details of naming here.